shortex.tex

```diff
@@ -1,188 +1,622 @@
+
+\bcent
+\btabr{@{}ll@{}}
+\toprule
+Comment Type & Syntax \\ \midrule
+remark & \verb!\RMK{Example remark}!\\ 
+margin remark & \verb!\mRMK{Example margin remark}!\\ 
+problem & \verb!\PRB{Example problem}!\\ 
+margin problem & \verb!\mPRB{Example margin problem}!\\ 
+\bottomrule
+\etabr
+\ecent
+
+Here is an example of how these look in a typical paragraph:\\
+\\
+
+Lorem ipsum dolor sit amet \RMK{Here is an inline remark}, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+nostrud exercitation \PRB{Here is an inline problem} ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in \mRMK{Here is a margin remark} voluptate velit esse cillum dolore eu
+fugiat nulla pariatur. Excepteur sint occaecat \mPRB{Here is a margin problem} cupidatat non proident, sunt...
+
+
+
+
+\subsection{Sets and set operations}
+\bcent
+\btabr{@{}lll@{}}
+\toprule
+Name & Syntax & Symbol  \\ \midrule
+reals	& \verb!\reals! & $\reals$ \\
+extended reals	& \verb!\extreals! & $\extreals$ \\
+rationals & \verb!\rats! & $\rats$\\
+integers	& \verb!\ints! & $\ints$ \\
+natural numbers	& \verb!\nats! & $\nats$ \\
+complex numbers	& \verb!\comps! & $\comps$ \\
+measures & \verb!\measures! & $\measures$\\
+probability measures & \verb!\pmeasures! & $\pmeasures$\\
+(big) intersection & \verb!\intersect!, \verb!\Intersect! & $\intersect$, $\Intersect$\\
+(big) union & \verb!\union!, \verb!\Union! & $\union$, $\Union$\\
+(big) disjoint union & \verb!\djunion!, \verb!\djUnion! & $\djunion$, $\djUnion$\\
+volume	& \verb!\vol! & $\vol$ \\
+diameter	& \verb!\diam! & $\diam$ \\
+boundary	& \verb!\boundary! & $\boundary$ \\
+closure	& \verb!\closure! & $\closure$ \\
+span	& \verb!\spann! & $\spann$ \\
+cone	& \verb!\cone! & $\cone$ \\
+convex hull	& \verb!\conv! & $\conv$ \\
+\bottomrule
+\etabr
+\ecent
+
+\subsection{Linear algebra}
+
+\bcent
+\btabr{@{}lll@{}}
+\toprule
+Name & Syntax & Symbol  \\ \midrule
+trace	& \verb!\tr A! & $\tr A$ \\
+rank	& \verb!\rank A! & $\rank A$ \\
+transpose	& \verb!A\T! & $A\T$ \\
+inverse transpose	& \verb!A\nT! & $A\nT$ \\
+diagonal	& \verb!\diag A! & $\diag A$ \\
+adjoint	& \verb!A\adj! & $A\adj$ \\
+spectrum	& \verb!\spec A! & $\spec A$ \\
+kronecker product & \verb!A\kron B! & $A\kron B$\\
+\bottomrule
+\etabr
+\ecent
+
+\subsection{Calculus}
+
+\bcent
+\renewcommand{\arraystretch}{1.5}
+\btabr{@{}lll@{}}
+\toprule
+Name & Syntax & Symbol  \\ \midrule
+differential symbol	& \verb!\dee x! & $\dee x$ \\
+gradient symbol	& \verb!\grad f! & $\grad f$ \\
+derivative	& \verb!\der{x}{y}! & $\der{x}{y}$ \\
+double derivative	& \verb!\dder{x}{y}! & $\dder{x}{y}$ \\
+derivative w.r.t.	& \verb!\derwrt{y}! & $\derwrt{y}$ \\
+partial derivative	& \verb!\pder{x}{y}! & $\pder{x}{y}$ \\
+partial double derivative	& \verb!\pdder{x}{y}! & $\pdder{x}{y}$ \\
+partial derivative coordinate & \verb!\pderi{x}{y}{i}! & $\pderi{x}{y}{i}$ \\
```

Comment:
this is the i-th partial derivative, not the derivative of the i-th coordinate

Reply:
done